test-lib.sh

```diff
@@ -430,6 +430,32 @@ ct_registry_from_os() {
   echo "$registry"
 }
 
+ # ct_get_public_image_name OS BASE_IMAGE_NAME VERSION
+# ----------------
+# Transform the arguments into public image name
+# Argument: OS - string containing the os version
+# Argument: BASE_IMAGE_NAME - string containing the base name of the image as defined in the Makefile
+# Argument: VERSION - string containing the version of the image as defined in the Makefile
+ct_get_public_image_name() {
```

Comment:
And of course. It is a good candidate to write a simple test for each `OS`.
Only this is missing and test is simple.

Please the test here https://github.com/sclorg/container-common-scripts/tree/master/tests/test-lib and call it from 
https://github.com/sclorg/container-common-scripts/blob/master/Makefile#L15

the test can be called `get_public_image_name`.